### PR TITLE
Rename enum tables

### PR DESCRIPTION
**Pull request by ttung**
### Description
@neevor suggested we do this in some PR but I forgot to do it before I committed it.

Depends on #60. #72 is needed to autogen the migration script.

#### Issue
[ch64743](https://app.clubhouse.io/genepi/stories/space/64743)

### Test plan
upgraded and downgraded from this schema

### CODE REVIEW

File: database_migrations/versions/20210209_134758_rename_enumtables.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+Revision ID: 20210209_134758
+Revises: 20210205_105007
+Create Date: 2021-02-09 13:47:59.276384
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
```
Comment (phoenixAja) [inline database_migrations/versions/20210209_134758_rename_enumtables.py:12]: idt we need this comment

File: database_migrations/versions/20210209_134758_rename_enumtables.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        referent_schema="aspen",
+    )
+    op.drop_constraint(
+        "fk_public_repository_entity_type_publicrepositorytypes",
+        "public_repository",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_public_repository_entity_type_public_repository_types"),
```
Comment (phoenixAja) [inline database_migrations/versions/20210209_134758_rename_enumtables.py:111]: these are really long names! would be nice to simplify some of these in the future if possible.

File: database_migrations/versions/20210209_134758_rename_enumtables.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+        referent_schema="aspen",
+    )
+    op.drop_constraint(
+        "fk_workflows_workflow_type_workflowtypes",
+        "workflows",
+        schema="aspen",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        op.f("fk_workflows_workflow_type_workflow_types"),
```
Comment (phoenixAja) [inline database_migrations/versions/20210209_134758_rename_enumtables.py:126]: this is a really confusing/ redundant name as well

Comment (phoenixAja) [reply database_migrations/versions/20210209_134758_rename_enumtables.py:126]: is it common practice to name foreign keys like this? Maybe I'm unfamiliar with how these are usually named.

**Review by phoenixAja — APPROVED** [review]

File: database_migrations/versions/20210209_134758_rename_enumtables.py (hunk 1 above)
Comment (ttung) [reply database_migrations/versions/20210209_134758_rename_enumtables.py:12]: It's part of the template. 🤷

File: database_migrations/versions/20210209_134758_rename_enumtables.py (hunk 2 above)
Comment (ttung) [reply database_migrations/versions/20210209_134758_rename_enumtables.py:111]: It's the default scheme used by sqlalchemy.

**Pull request merged**